src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCaseEquivalences.cs

```diff
@@ -131,34 +125,35 @@ static bool IsTurkishOrAzeri(ReadOnlySpan<char> cultureName)
         /// equivalent in a case comparison for 'A'.
         /// </remarks>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static bool TryFindCaseEquivalencesForChar(char c, out ReadOnlySpan<char> equivalences)
+        private static bool TryFindCaseEquivalencesForChar(char c, out ReadOnlySpan<char> equivalences)
         {
-            // Right shifting by 10 in order to divide by 1024 (CharactersPerRange)
-            Debug.Assert((c >> 10) < 0xFF);
-            byte index = (byte)(c >> 10);
-            ushort FirstLevelLookupValue = EquivalenceFirstLevelLookup[index];
+            // Dividing by 1024 whih represent the number of characters per range, in order to get the range index for c
```

Comment:
"Dividing by 1024 whih represent" => "Dividing by 1024, which represents"

Also, does this 1024 need to be declared as a const somewhere both this code and the generator code can share it?